KonyAccelerate/controllers/mobile/frmFeedbackController/frmFeedbackController.js

```diff
@@ -0,0 +1,131 @@
+define({
+
+    //Type your controller code here 
+    selectedImage: "starselected.png",
+    unselectedImage: "starunselected.png",
+    objectServiceL: "KonyAccelerateStorageService",
+    dataModelObject: "opinion_answers",
+    ratingQuestionId: "11",
+    feedbackQuestionId: "12",
+    opinion_id: 18,
+    /**
+     * @function onClickOfRatingPrompt
+     * @scope private
+     * @description this function is invoked to toggle the selected and unselected images.
+     * @param eventObject - contains the info of the widget which is clicked.
+     */
+    onClickOfRatingPrompt: function(eventObject) {
+        var id = eventObject.id;
+        var index = parseInt(id[id.length - 1]);
+        this.index = index + 1;
+        for (var iterate = 0; iterate <= 4; iterate++) {
+            if (iterate <= index) {
+                this.view["imgRating" + iterate].src = this.selectedImage;
+            } else {
+                this.view["imgRating" + iterate].src = this.unselectedImage;
+            }
+        }
+
+    },
+    /**
+     * @function onClickOfRatingPrompt
+     * @scope private
+     * @description this function is invoked to toggle the selected and unselected images.
+     * @param eventObject - contains the info of the widget which is clicked.
+     */
+    onClickOfSubmitButton: function() {
+        var currentUserInfo=kony.store.getItem("currentUserInfo");
+      	var userId=currentUserInfo.user_id;
+        if (kony.sdk.isNullOrUndefined(this.index)) {
+            this.index = 0;
+        }
+        var feedback = this.view.txtAreaComments.text;
+        var feedbacks = [];
+        var rating = {
+            "userId": userId,
+            "opinion_question_id": this.feedbackQuestionId,
+            "response_text": feedback,
+            "opinion_id": this.opinion_id,
+        };
+        var feedbackComment = {
+            "userId": userId,
+            "opinion_question_id": this.ratingQuestionId,
+            "opinion_id": this.opinion_id,
+            "response_text": this.index,
+        };
+        feedbacks.push(rating);
+        feedbacks.push(feedbackComment);
+        var batch = {
+            "records": feedbacks
+        };
+        this.createRecord(this.objectServiceL, this.dataModelObject, batch, this.successInStoreFeedback.bind(this), this.failInStoreFeedback.bind(this));
+
+    },
+    /**
```

Comment:
Please move the createRecord function to the "obejctServiceOperation.js" file which is a global module.

Reply:
For the scope of the function, according to the standard, we follow there is no need for mentioning scope separately. Just add "@Private" and move that to the last.

Reply:
Moved to Global module